app.js

```diff
@@ -0,0 +1,37 @@
+const express  = require('express');
+const bodyParser = require('body-parser');
```

Comment:
I think bodyparser is depreciated and express have added the functionality in built